crates/optimism/node/src/args.rs

```diff
@@ -17,6 +17,10 @@ pub struct RollupArgs {
     #[arg(long = "rollup.disable-tx-pool-gossip")]
     pub disable_txpool_gossip: bool,
 
+    /// Disable transaction pool admission
+    #[arg(long = "rollup.disable-tx-pool-admission")]
+    pub disable_txpool_admisson: bool,
```

Comment:
I checked op-geth again, and we need to inverse this, and add this the same way op-geth does:

https://github.com/ethereum-optimism/op-geth/blob/36501a7023fd85f3492a1af6f1474a0113bb83fe/cmd/utils/flags.go#L947-L951